Paralayout/FlexibleDistributionProxy.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Copyright © 2025 Nick Entin
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+@MainActor
+public final class FlexibleDistributionProxy: ViewDistributionSpecifying {
+
+    // MARK: Initialization
+
+    public init(weight: CGFloat) {
```

Comment:
Weight of `3` is effectively equivalent to `3.flexible` in a distribution?

I'm struggling to understand why we need a new concept here. Can we reuse a `ViewDistribution` here?

Reply:
Yep, weight here means the same as the weight of a flexible spacer. So a flexible proxy with weight of 1 and flexible spacer with weight of 1 will be allocated equal space in a distribution. Given a flexible proxy with weight of 2 and flexible spacer with weight of 1, the proxy will be allocated twice the space of the spacer. Etc.

I'm considering also making a fixed-size proxy, but I need to think through the API a bit more.